charts/karpenter/templates/deployment.yaml

```diff
@@ -88,6 +88,10 @@ spec:
             - name: LOG_LEVEL
               value: "{{ . }}"
           {{- end }}
+            - name: LOG_OUTPUT_PATHS
+              value: {{ include "karpenter.outputPathsList" . }}
+            - name: LOG_ERROR_OUTPUT_PATHS
+              value: {{ include "karpenter.errorOutputPathsList" . }}
```

Comment:
Can we just call the join here if we can just do this without a helper?

Reply:
Yep, I'll make the fix.